src/server/src/main/java/io/cassandrareaper/resources/RepairRunResource.java

```diff
@@ -104,20 +104,23 @@ public Response addRepairRun(
       @QueryParam("incrementalRepair") Optional<String> incrementalRepairStr,
       @QueryParam("nodes") Optional<String> nodesToRepairParam,
       @QueryParam("datacenters") Optional<String> datacentersToRepairParam,
-      @QueryParam("blacklistedTables") Optional<String> blacklistedTableNamesParam) {
+      @QueryParam("blacklistedTables") Optional<String> blacklistedTableNamesParam,
+      @QueryParam("repairThreadCount") Optional<Integer> repairThreadCountParam) {
 
     try {
-      final Response possibleFailedResponse = RepairRunResource.checkRequestForAddRepair(
-          context,
-          clusterName,
-          keyspace,
-          owner,
-          segmentCountPerNode,
-          repairParallelism,
-          intensityStr,
-          incrementalRepairStr,
-          nodesToRepairParam,
-          datacentersToRepairParam);
+      final Response possibleFailedResponse =
```

Comment:
nit-pick: the operator shall carry over to the new line. 

ie
```
     = RepairRunResource.checkRequestForAddRepair(
              context,
              clusterName,
              keyspace,
              owner,
              segmentCountPerNode,
              repairParallelism,
              intensityStr,
              incrementalRepairStr,
              nodesToRepairParam,
              datacentersToRepairParam,
              repairThreadCountParam);
```
it's an axiom to make the distinction between assignment, method calls, and arguments, clearer.